metrics/crypto/backend_test.go

```diff
@@ -9,14 +9,16 @@ type testBackend struct {
 	panicSent bool
```

Comment:
In the name of the commit, please clarify what kind of fix is that?

Reply:
fixed